File: FastClass/salida/php/test5.php
<?php 
class test5 { 
	public $readwrite;
	public $writeconst;
	public $readcont;
	public $leescrcom;

	public __construct($writeconst, $readcont, $leescrcom) {
		$this->writeconst = $writeconst;
		$this->readcont = $readcont;
		$this->leescrcom = $leescrcom;
	}
	public function getReadwrite(){
		return $this->readwrite;
	}
	public function setReadwrite($readwrite){
		$this->readwrite = $readwrite;
	}
	public function setWriteconst($writeconst){
		$this->writeconst = $writeconst;
	}
	public function getReadcont(){
		return $this->readcont;
	}
	public function getLeescrcom(){
		return $this->leescrcom;
	}
	public function setLeescrcom($leescrcom){
		$this->leescrcom = $leescrcom;
	}

}
?>

File: FastClass/salida/java/test5.java
public class test5 { 
	int readwrite;
	int writeconst;
	int readcont;
	int leescrcom;

	public test5(int writeconst, int readcont, int leescrcom) {
		this.writeconst = writeconst;
		this.readcont = readcont;
		this.leescrcom = leescrcom;
	}
	public int getReadwrite(){
		return this.readwrite;
	}
	public void setReadwrite(int readwrite){
		this.readwrite = readwrite;
	}
	public void setWriteconst(int writeconst){
		this.writeconst = writeconst;
	}
	public int getReadcont(){
		return this.readcont;
	}
	public int getLeescrcom(){
		return this.leescrcom;
	}
	public void setLeescrcom(int leescrcom){
		this.leescrcom = leescrcom;
	}

}
